.github/BOTMETA.yml

```diff
@@ -302,6 +302,7 @@ files:
   $modules/source_control/git.py: $team_ansible
   $modules/source_control/github_key.py:
     ignored: erydo
+  $modules/storage/hpe3par/: farhan7500 gautamphegde
```

Comment:
Just a suggestion. Can we keep this as $team_hpe?
Similar thing is done for $team_netapp at line number 307